cirq/aqt/aqt_device.py

```diff
@@ -0,0 +1,150 @@
+"""Current device parameters for the AQT/UIBK ion trap device
+
+The device is based on a linear calcium ion string with
+arbitrary connectivity. For more information see:
+
+https://quantumoptics.at/en/publications/journal-articles.html
+
+https://iopscience.iop.org/article/10.1088/1367-2630/15/12/123012/meta
+
+The native gate set consists of the local gates: X,Y, and XX entangling gates
+
+"""
+import json
+from typing import Union, Tuple, List, Sequence
+from cirq import Circuit, ops, LineQubit, study, IonDevice, DensityMatrixSimulator, NoiseModel, NO_NOISE
+from cirq import measure, X, Y, XX, Duration, depolarize
+import cirq
+
+gate_dict = {'X': X, 'Y': Y, 'MS': XX}
+
+
+def get_op_string(op_obj: ops.EigenGate):
+    """Find the string representation for a given gate
+    Params:
+        op_obj: Gate object, out of: XXPowGate, XPowGate, YPowGate"""
+    try:
+        op_obj = op_obj._gate
+    except AttributeError:
```

Comment:
Don't just catch an exception and drop it like this. Use `ops.op_gate_of_type` instead.